src/theme/index.less

```
@@ -23,6 +23,11 @@
   /* 2024-12-26 v8 版本解决图表 Tooltip 容器会撑开页面导致滚动条出现的问题 */
   body {
     overflow: hidden;
+    font-family: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)